library/src/main/java/com/google/android/exoplayer/text/eia608/ClosedCaptionCtrl.java

```diff
@@ -15,6 +15,9 @@
  */
 package com.google.android.exoplayer.text.eia608;
 
+import android.graphics.Color;
+import android.util.Log;
```

Comment:
This is the sign that I already removed most of the debugging before submitting the code ;)